test/factories/billing/usage/trackers.rb

```diff
@@ -3,5 +3,6 @@
     duration { 1 }
     interval { "month" }
     association :team
+    association :trackable, factory: :team
```

Comment:
Do we need to somehow use `BulletTrain::Billing::Usage.parent_association` here in case somebody has changed the default?